  File "feature_extract.py", line 149
    if opt.method == 'graphvlad':
    ^
IndentationError: unexpected indent
  File "feature_extract.py", line 149
    if opt.method == 'graphvlad':
    ^
IndentationError: unexpected indent
Traceback (most recent call last):
  File "feature_match.py", line 215, in <module>
    main()
  File "feature_match.py", line 207, in main
    feature_match(dataset, device, opt, config)
  File "feature_match.py", line 107, in feature_match
    qFeat = np.load(input_query_global_features_prefix)
  File "/home/m.maqboolbhutta/.conda/envs/patchnetvlad/lib/python3.8/site-packages/numpy/lib/npyio.py", line 417, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '/home/m.maqboolbhutta/usman_ws/models/features/mapillarysf_query/globalfeats.npy'
  File "feature_extract.py", line 149
    if opt.method == 'graphvlad':
    ^
IndentationError: unexpected indent
  File "feature_extract.py", line 149
    if opt.method == 'graphvlad':
    ^
IndentationError: unexpected indent
Traceback (most recent call last):
  File "feature_match.py", line 215, in <module>
    main()
  File "feature_match.py", line 207, in main
    feature_match(dataset, device, opt, config)
  File "feature_match.py", line 107, in feature_match
    qFeat = np.load(input_query_global_features_prefix)
  File "/home/m.maqboolbhutta/.conda/envs/patchnetvlad/lib/python3.8/site-packages/numpy/lib/npyio.py", line 417, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '/home/m.maqboolbhutta/usman_ws/models/features/mapillarycph_query/globalfeats.npy'
/blue/hmedeiros/m.maqboolbhutta/codes/mapillary_sls/mapillary_sls/datasets/msls.py:217: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  self.pIdx = np.asarray(self.pIdx)
slurmstepd: error: *** JOB 24888964 ON c0801a-s23 CANCELLED AT 2024-02-29T14:17:48 ***
